src/mscorlib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -7,6 +7,27 @@ namespace System.Globalization
     internal sealed partial class GlobalizationMode
     {
         private const string c_InvariantModeConfigSwitch = "System.Globalization.Invariant";
+        // Linux doesn't support environment variable names including dots
+        private const string c_InvariantModeEnvironmentVariable = "DOTNET_System_Globalization_Invariant";
         internal static bool Invariant { get; } = GetGlobalizationInvariantMode();
+
+        // GetInvariantSwitchValue calls CLRConfig first to detect if the switch is defined in the config file.
+        // if the switch is defined we just use the value of this switch. otherwise, we'll try to get the switch
+        // value from the environment variable if it is defined.
+        internal static bool GetInvariantSwitchValue()
+        {
+            bool exist;
+            bool ret = CLRConfig.GetBoolValue(c_InvariantModeConfigSwitch, out exist);
+            if (!exist)
+            {
+                string switchValue = Environment.GetEnvironmentVariable(c_InvariantModeEnvironmentVariable);
+                if (switchValue != null)
+                {
+                    ret = switchValue.Equals("true", StringComparison.OrdinalIgnoreCase);
```

Comment:
It is more common to use 0/1 for these. (We can recognize true/false as well, but 0/1 has to work.)